### PR TITLE
Replacement of spacers in LazyLists and LazyGrids

### PR DESCRIPTION
**Pull request by chikoski**
This pull request will remove the Spacers in the LazyLists and LazyGrids to set margins between the lists / grids and other components by specifying the contentPadding parameter to the scrollable containers. 

This PR also updates the dependencies.

### CODE REVIEW

File: JetStreamCompose/gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,34 +1,33 @@
...
 activity-compose = "1.8.2"
-android-gradle-plugin = "8.2.1"
-android-test-plugin = "8.2.1"
-benchmark-macro-junit4 = "1.2.2"
+android-gradle-plugin = "8.2.2"
+android-test-plugin = "8.2.2"
+benchmark-macro-junit4 = "1.2.3"
 coil-compose = "2.5.0"
-compose-bom = "2023.10.01"
+compose-bom = "2024.02.02"
```
Comment (mlykotom) [inline JetStreamCompose/gradle/libs.versions.toml:7]:
```suggestion
compose-bom = "2024.03.00"
```

File: JetStreamCompose/gradle/libs.versions.toml [hunk 2]
```diff
@@ -1,34 +1,33 @@
 [versions]
 activity-compose = "1.8.2"
-android-gradle-plugin = "8.2.1"
-android-test-plugin = "8.2.1"
-benchmark-macro-junit4 = "1.2.2"
+android-gradle-plugin = "8.2.2"
+android-test-plugin = "8.2.2"
```
Comment (mlykotom) [inline JetStreamCompose/gradle/libs.versions.toml:4]:
```suggestion
android-gradle-plugin = "8.3.1"
android-test-plugin = "8.3.1"
```

File: JetStreamCompose/gradle/libs.versions.toml [hunk 3]
```diff
@@ -1,34 +1,33 @@
...
-compose-bom = "2023.10.01"
+compose-bom = "2024.02.02"
 compose-for-tv = "1.0.0-alpha10"
-compose-ui = "1.6.0-rc01"
 core-ktx = "1.12.0"
 core-splashscreen = "1.0.1"
-hilt-navigation-compose = "1.1.0"
-hilt-android = "2.48.1"
+hilt-navigation-compose = "1.2.0"
+hilt-android = "2.49"
```
Comment (mlykotom) [inline JetStreamCompose/gradle/libs.versions.toml:12]:
```suggestion
hilt-android = "2.51"
```

**Review by mlykotom — APPROVED** [review]
Added suggestions for newer stable versions, otherwise LGTM!

**Comment by chikoski** [comment]
Thanks for quick response! 

Updated the version catalog with the suggested ones.

**Pull request merged**